### DISCUSSION TITLE
I have installed Blockscout but Application Inders exits and the explorer doesn't update itself to the latest chain data

### DISCUSSION THREAD

**Discussion by wahdatw** [blockscout/blockscout#6719] [General]
Here is the log. 

Can someone please help me resolve this issue? 

-- Logs begin at Sat 2023-01-14 08:47:48 UTC. --
Jan 16 01:48:19 vultr mix[123739]: @𔘧: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @𔘦: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @𔘠: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @𔘟: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @䨌: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @䨊: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @䨉: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: @䨇: (-32601) the method trace_block does not exist/is not available
Jan 16 01:48:19 vultr mix[123739]: 2023-01-16T01:48:19.346 application=indexer fetcher=internal_transaction count=10 error_count=10 [error] failed to fetch internal transactions for blocks: [%{code: -32601, data: %{"blockNumber" => 80967}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 80965}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 80962}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 80966}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 80961}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 80960}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 79428}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 79436}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 79434}, message: "the method trace_replayBlockTransactions does not exist/is not available"}, %{code: -32601, data: %{"blockNumber" => 79431}, message: "the method trace_replayBlockTransactions does not exist/is not available"}]
Jan 16 01:48:19 vultr mix[123739]: 2023-01-16T01:48:19.399 [notice] Application indexer exited: shutdown

**Comment by AzimAnuar** [comment]
yeah me too.

**Comment by wahdatw** [comment]
I solved this.

On Mon, Jan 30, 2023 at 8:44 PM Azim Anuar ***@***.***> wrote:

> yeah me too.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blockscout/blockscout/discussions/6719#discussioncomment-4822637>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMBKYIN2YJBV6WU2HHMOE3TWVAK3DANCNFSM6AAAAAAT4GLVVI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
M. Wahdat Wassel
017657959262
